anvil/src/input_handler.rs

```diff
@@ -1306,6 +1306,7 @@ impl AnvilState<UdevData> {
 }
 
 /// Possible results of a keyboard action
+#[allow(dead_code)] // some of these are only read if udev is enabled
```

Comment:
Are they always written to? Why not feature gate the variants behind the backend?

Reply:
Yes the corresponding key binding will still generate the KeyAction. Of course we could feature-gate all of that as well, but I am not sure that is cleaner than this short line.